copier/config/factory.py

```diff
@@ -65,7 +67,8 @@ def make_config(
     subdirectory: OptStr = None,
     **kwargs,
 ) -> ConfigData:
-    """Provides the configuration object, merged from the different sources.
+    """
+    Provides the configuration object, merged from the different sources.
```

Comment:
According to https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings this was OK:

```suggestion
    """Provides the configuration object, merged from the different sources.
```